dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -431,6 +435,37 @@ public Workflow updateWorkflow(@ApiParam(hidden = true) @Auth User user,
     @PUT
     @Timed
     @UnitOfWork
+    @Path("/{workflowId}/verify")
+    @RolesAllowed("admin")
+    @ApiOperation(value = "Verify or unverify a workflow. ADMIN ONLY", response = Workflow.class)
+    public Workflow verifyWorkflow(@ApiParam(hidden = true) @Auth User user,
+            @ApiParam(value = "Workflow to modify.", required = true) @PathParam("workflowId") Long workflowId,
+            @ApiParam(value = "Object containing verification information.", required = true) VerifyRequest verifyRequest) {
+        Workflow workflow = workflowDAO.findById(workflowId);
+        Helper.checkEntry(workflow);
+        // Note: if you set someone as an admin, they are not actually admin right away. Users must wait until after the
```

Comment:
Add this as an issue or a reply to the issue where you documented the problem initially so that this problem is searchable.